marmot.go

```diff
@@ -44,6 +46,22 @@ func main() {
 		log.Logger = gLog.Level(zerolog.InfoLevel)
 	}
 
+	if *cfg.ProfServer != "" {
+		go func() {
+			mux := http.NewServeMux()
+			mux.HandleFunc("/debug/pprof/", pprof.Index)
```

Comment:
Is `/debug` prefix a standard practice? Since we are already launching a dedicated binding why do we need additional prefixes?